README.md

```diff
@@ -4,36 +4,28 @@
 ![Tensorflow](https://img.shields.io/badge/tensorflow-v2.9.0+-success.svg)
 [![Contributions Welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/keras-team/keras-cv/issues)
 
-# Vision
-A computer vision library dedicated for auto-driving, robotics and on device applications.
 
 # Mission
 
-KerasCV is a layered repository consisting of core components and modeling components.
+KerasCV is a computer vision library of modular computer vision oriented Keras components.
```

Comment:
It might be cool if we could align the first sentences of KerasCV's and KerasNLP's mission. For KerasNLP we have -> "KerasNLP is a natural language processing library that supports users through their entire development cycle." Def open to changing that if we find some language we think will be exciting for readers and applicable for both projects.

cc @jbischof

Reply:
Open to feedback.  I think the CV selling point is a bit different in that its based on modularity - but open to feedback.

Reply:
Modular only makes our second sentence currently :P, but I would be totally down to bump it up. Totally fine to leave both as is if this is feeling too tricky, was just thinking it might be a good way to show some alignment between the packages.